tests/util/permutation_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
No test coverage of invalid size range

Reply:
I added a test and also for the conversion from proto.

Reply:
Thanks!